pkg/limayaml/limayaml.go

```diff
@@ -43,6 +45,10 @@ const (
 
 	REVSSHFS MountType = "reverse-sshfs"
 	NINEP    MountType = "9p"
+	VIRTIOFS MountType = "virtiofs"
```

Comment:
Please explain this in https://github.com/lima-vm/lima/blob/master/docs/mount.md

Reply:
Done